src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -205,3 +224,71 @@ DeparseAlterDatabaseSetStmt(Node *node)
 
 	return str.data;
 }
+
+
+static void
+AppendCreateDatabaseStmt(StringInfo buf, CreatedbStmt *stmt)
+{
+	appendStringInfo(buf,
+					 "CREATE DATABASE %s",
+					 quote_identifier(stmt->dbname));
+
+	DefElem *option = NULL;
+
+	foreach_ptr(option, stmt->options)
+	{
+		optionToStatement(buf, option, create_database_option_formats, lengthof(
+							  create_database_option_formats));
+	}
```

Comment:
I wonder should we also add options that are not provided in input DDL based on DEFAULT values applicable for this server, say, for the options where it's possible that workers could have picked a different DEFAULT value than this server for some options.